_data/toc.yaml

```diff
@@ -65,6 +65,42 @@ guides:
     - path: /edge/engine/reference/commandline/docker/
       title: Docker Edge CLI reference
       nosync: true
+  - sectiontitle: Docker for IBM Cloud (Beta)
+    section:
+    - path: /docker-for-ibm-cloud/why/
+      title: Why Docker for IBM Cloud?
+    - path: /docker-for-ibm-cloud/quickstart/
+      title: Quick Start
+    - path: /docker-for-ibm-cloud/
+      title: "Setup & prerequisites"
```

Comment:
No quotes (maybe it works but we don't use it elsewhere)

Reply:
Joao had said to include those because of the ampersand?